src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -1319,12 +1320,12 @@ private void VariableSummaries(ResourceVariable var)
 
         }
 
-        private static TensorFlowSessionWrapper LoadTensorFlowSessionFromMetaGraph(IHostEnvironment env, Architecture arch)
+        private TensorFlowSessionWrapper LoadTensorFlowSessionFromMetaGraph(IHostEnvironment env, Architecture arch)
         {
             var modelFileName = ModelFileName[arch];
-            var modelFilePath = Path.Combine(_resourcePath, modelFileName);
+            var modelFilePath = Path.Combine(_resourcePath, "MLNET", modelFileName);
```

Comment:
>M [](http://example.com/codeflow?start=61&length=1)

Doesn't `_resourcePath` already have `MLNET` appended to it, from the constructor?

Reply:
It did yeah, and it is what was failing the tests. That has been resolved now.